### PR TITLE
Feature/461 seed db with placeholder content

### PR DESCRIPTION
**Pull request by kingdomlevel**
The work done in this PR is designed to seed the initial database using Django database migrations, using [this process](https://stackoverflow.com/questions/25960850/loading-initial-data-with-django-1-7-and-data-migrations#25981899) as recommended by @joel-stobart ([on slack](https://snookteam.slack.com/archives/G01QHLZ6YSV/p1618407865105100)).

This PR:

- Adds some sample data to seed the DB (thanks @adamgparsons !)
- adds a DB migration to load this data in as part of the `migrate` step.

To run:
1. Apply the migrations:`python manage.py migrate`
2. Inspect in the db: `python manage.py dbshell`.

After following these steps you should be able to see that the data from the `fixtures` is populating your `psql` db:
image

### CODE REVIEW

**Comment by adamgparsons** [comment]
Looks good from my side!

File: CyberHealth/assessment/migrations/0004_load_intial_data.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            return apps.get_model(model_identifier)
+        except (LookupError, TypeError):
+            raise base.DeserializationError(
+                "Invalid model identifier: '%s'" % model_identifier)
+    # Replace the _get_model() function on the module, so loaddata can utilize it.
+    python._get_model = _get_model
+    try:
+        # Call loaddata command
+        fixture_files = list(pathlib.Path().glob('*/fixtures/*.json'))
+        # this bit need fixed!!!
```
Comment (kingdomlevel) [inline CyberHealth/assessment/migrations/0004_load_intial_data.py:28]: this comment can come out

Comment (joel-stobart) [reply CyberHealth/assessment/migrations/0004_load_intial_data.py:28]: That would be great :)

**Review by joel-stobart — CHANGES_REQUESTED** [review]
I like the simplicity of this commit. Can you just confirm that it does the thing

**Review by h0usemaintenance — APPROVED** [review]
Files look ok and python manage.py migrate loads the expected data using the models created.

**Review by joel-stobart — APPROVED** [review]

**Pull request merged**